Bottom: 15a60afddbcd2bcb8f38b4e79eb5bd3238ec4121
Top:    f7665070eee7dad682c00e546bb19fb28242e431
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 00:39:54 +0000

Refresh of hid-gtk-gl-start-with-the-sten

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 4573b10..409dcac 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1213,6 +1213,11 @@ ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int dept
   glClear (GL_COLOR_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
   hidgl_reset_stencil_usage ();
 
+  /* Disable the stencil test until we need it - otherwise it gets dirty */
+  glDisable (GL_STENCIL_TEST);
+  glStencilMask (0);
+  glStencilFunc (GL_ALWAYS, 0, 0);
+
   /* call the drawing routine */
   ghid_invalidate_current_gc ();
   glPushMatrix ();
